env/README.md

```diff
@@ -0,0 +1,46 @@
+# Setting up environments for The Data Science Community of Practice scipts/notebooks
+
+The environments for code found in this repository, either in scripts or in notebooks, depending on the platform and type of work you are doing, are all primarily provided by conda environments.
+
+To create an environment from any of the *.yml* files, run the following command
+```bash
+conda env create --file requirement.yml
+```
+where you replace `requirements.yml` with the particular requirements file in the folder required for the files you want to run. There several different environments as different packages are required for different activities or demonstrations, some of which are mutually incompatible or make for slow/difficult to resolve installations, so we have tried to keep each environment as simple as possible to improve performance, as well as fixing compatible major and minor package versions to help environment solving.
+
+**The installation of environments with the yml files can take a significant amount of memory, for this reason we present two methods to overcome this issue: Use mamba to install yml files, or Install the provided lock files which are system dependant.**
+
+#### Installing environments via mamba
+
+To create an environment using mamba, run the following set of commands: <br>
+First, using the desktop conda, create an environment that contains mamba:
+```bash
+conda create --name mamba --channel conda-forge mamba
+```
+Now activate that environment:
+```bash
+conda activate mamba
+```
+Now create your data science environment:
+```bash
+mamba env create --file requirements.yml
+```
+
+#### Installing environments via provided lock files
+
+To create an environment from any of the *.lock* files, run the following command which also requires a name for the environment to be specified
+```bash
+conda create --name myenv --file requirements.lock
+```
+Generation of the lock files was performed in the Met Office VDI, and will fail to install on systems that are significantly different<br>
```

Comment:
We might have other lock files for other environments in future. So maybe we could rephrase this to something like:
```
Lock file are generated on and for specific systems, and will fail to install on systems that are significantly different from the target. Each lock file includes a reference to the target system as follows:
* mo_linux - Met Office internal linux "desktop" systems
```